less/pages/encrypt/components.less

```diff
@@ -177,6 +177,7 @@ video{
   position: relative;
   margin: 5px;
   flex: 1 1 40%;
+  min-width: 0;
```

Comment:
This works, but I recently fixed a similar issue here: https://github.com/mozilla/teach.mozilla.org/commit/99672d323b35f2caf3b22b0f205f5319abb03080

I found the `max-width: 100%` + flexbox + firefox stable might be doing weird things. This might be your issue too. Changing the `max-width: 100%;` in base.less on `img, video {}` to `width: 100%;` might be a better fix, but, it's not a huge deal and both solutions would get an r+ from me.